src/software/jetson_nano/lcd_user_interface/lcd_user_interface.py

```diff
@@ -0,0 +1,97 @@
+import digitalio
+import board
+from PIL import Image, ImageDraw, ImageOps
+import adafruit_rgb_display.st7735 as st7735
+
+"""
+Information for the display that we are using can be found here: https://www.adafruit.com/product/358#description
+
+Pin Connections:
+    Jetson Nano              LCD Display
+        Pin 6  (GND)            Pin 1  (GND)
+        Pin 1  (3.3V)           Pin 2  (VCC)
+        Pin 18                  Pin 3  (Reset)
+        Pin 22                  Pin 4  (D/C) 
+        N/C                     Pin 5  (CARD_CS)
+        Pin 24 (CS)             Pin 6  (TFT_CS)
+        Pin 19 (MOSI)           Pin 7  (MOSI)
+        Pin 23 (SCK)            Pin 8  (SCK)
+        N/C                     Pin 9  (MISO)
+        Pin 1  (3.3V)           Pin 10 (LITE)
+"""
+
+# Configuration for CS and DC pins (these are PiTFT defaults):
+CS_PIN = digitalio.DigitalInOut(board.CE0)  # Pin 24
+DC_PIN = digitalio.DigitalInOut(board.D25)  # Pin 22
+RESET_PIN = digitalio.DigitalInOut(board.D24)  # Pin 18
+
+# Config for display baudrate (default max is 24mhz):
+BAUDRATE = 24000000
+# Config the proper screen rotation
+ROTATION = 90
+
+# Setup SPI bus using hardware SPI:
+SPI = board.SPI()
+
+# We will use RGB colour model
+COLOUR_MODEL = "RGB"
+
+
+class lcd_display:
+    def __init__(self):
+        """ Create a lcd_dislpay object """
+        # Create the display for 1.8" ST7735R:
+        self.disp = st7735.ST7735R(
+            SPI,
+            rotation=ROTATION,
+            cs=CS_PIN,
+            dc=DC_PIN,
+            rst=RESET_PIN,
+            baudrate=BAUDRATE,
+        )
+
+        # Set display dimensions.
+        # We swap height/width to rotate display to landscape
+        self.width = self.disp.height
+        self.height = self.disp.width
+
+        # Initialize to an empty black screen
+        self.clear_screen()
+
+    def clear_screen(self):
+        """ Clear this LCD display, make it black """
+        # Create blank image for drawing.
+        image = Image.new(COLOUR_MODEL, (self.width, self.height))
+
+        # Get drawing object to draw on image.
+        draw = ImageDraw.Draw(image)
+
+        # Draw a black filled box to clear the image.
+        draw.rectangle((0, 0, self.width, self.height), outline=0, fill=(0, 0, 0))
+        self.disp.image(image)
+
+    def draw_logo(self):
+        """ Draw Thunderbots Logo on this LCD display """
+        self.clear_screen()
+
+        image = Image.open("./imgs/tbots.jpg")
+        image = ImageOps.invert(image)  # Image has inverted colour
```

Comment:
is this function specific to our logo?

or can we rename it to `draw_image(self, path_to_image)`

and then pass in the path when we draw?

Reply:
done